erts/preloaded/src/erlang.erl

```diff
@@ -1210,12 +2595,36 @@ float_to_list(_Float, _Options) ->
 
 %% floor/1
 %% Shadowed by erl_bif_types: erlang:floor/1
+-doc """
+Returns the largest integer not greater than `Number`. For example:
+
+```erlang
+> floor(-10.5).
+-11
+```
+
+Allowed in guard tests.
+""".
+-doc(#{since => <<"OTP 20.0">>}).
 -spec floor(Number) -> integer() when
       Number :: number().
 floor(_) ->
     erlang:nif_error(undef).
 
 %% fun_info/2
+-doc """
+Returns information about `Fun` as specified by `Item`, in the form
+`{Item,Info}`.
+
+For any fun, `Item` can be any of the atoms `module`, `name`, `arity`, `env`, or
+`type`.
+
+For a local fun, `Item` can also be any of the atoms `index`, `new_index`,
+`new_uniq`, `uniq`, and `pid`. For an external fun, the value of any of these
+items is always the atom `undefined`.
+
+See [`erlang:fun_info/1`](`fun_info/1`).
```

Comment:
I'm assuming this markdown is automatically generated off XML docs but this could be:

```markdown
See [`erlang:fun_info/1`].
```

or

```markdown
See [`fun_info/1`].
```

Reply:
Yeah, the convertion tool tries to collapse this types of links, but it does not always work with prefixed links like that.